core/src/main/scala/org/apache/spark/scheduler/TaskResultGetter.scala

```diff
@@ -32,7 +32,7 @@ import org.apache.spark.util.{LongAccumulator, ThreadUtils, Utils}
 /**
  * Runs a thread pool that deserializes and remotely fetches (if necessary) task results.
  */
-private[spark] class TaskResultGetter(sparkEnv: SparkEnv, scheduler: TaskSchedulerImpl)
+private[spark] class TaskResultGetter(sparkEnv: SparkEnv, taskScheduler: TaskSchedulerImpl)
```

Comment:
ditto.

Reply:
Traditionally, the `scheduler` looks more minimalist here. No one would suspect that it is related to a task.